Sources/Tracking/Background Queue/QueueRunRequest.swift

```diff
@@ -35,14 +43,18 @@ public class CioQueueRunRequest: QueueRunRequest {
         runTasks(query: inventory, queryTotalNumberTasks: inventory.count)
     }
 
-    private func runTasks(query: [QueueTaskMetadata], queryTotalNumberTasks: Int) {
-        if query.isEmpty { // we hit the end of the current inventory. Done!
+    private func runTasks(
+        query: [QueueTaskMetadata],
```

Comment:
`query` here feels like a confusing name even moreso now with the filtering

Reply:
Agree. Renamed to `queueInventory`